CRM/Custom/Form/Field.php

```diff
@@ -441,21 +438,12 @@ public function buildQuickForm() {
       $attributes['help_post']
     );
 
-    // is active ?
+    $this->add('advcheckbox', 'is_required', ts('Required?'));
+    $this->addElement('advcheckbox', 'is_searchable', ts('Optimize for Search?'));
+    $this->addRadio('is_search_range', ts('Search by Range?'), [ts('No'), ts('Yes')]);
```

Comment:
While we're at it let's cleanup the trailing question marks?
We've been moving away from this for a while?
Because ending a field label with a question mark?
Is dumb?

Reply:
Couldn't help it:

![image](https://github.com/civicrm/civicrm-core/assets/254741/976e6455-f694-48ae-b046-46ba751abd67)

Reply:
@colemanw done